code/places/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val mavenPlacesVersion: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile"
+    enableSpotlessPrettierForJava = true
+    enableSpotless = true
+ 
+    publishing {
+        gitRepoName = "aepsdk-places-android"
+        addCoreDependency(mavenCoreVersion)
+    }
+}
+
+dependencies {
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
```

Comment:
We're missing edge dependency here

Reply:
Places does not have hard dependencies on edge. We can ignore them here